src/setupTests.js

```diff
@@ -6,3 +6,16 @@
  */
 
 global.IS_REACT_ACT_ENVIRONMENT = true;
+
+// jest.config.js
+module.exports = {
```

Comment:
why this here, we don't need this in gridstudy?

Reply:
I came across it while reading the documentation of Ag-Grid: <https://www.ag-grid.com/react-data-grid/testing/>